lib/reek/tree_walker.rb

```diff
@@ -94,6 +94,13 @@ def process_send(exp)
       if visibility_modifier? exp
         element.track_visibility(exp.method_name, exp.arg_names)
       end
+      if attribute_writer? exp
+        exp.args.each do |arg|
+          next unless arg.type == :sym
+          inside_new_context(Context::MethodContext, arg) do
+          end
```

Comment:
I assume that this is needed for `TreeWalker#inside_new_context`’s side-effects (appending the child context) – but the empty block looks strange.

I think `inside_new_context` should either `yield if block_given?` or there should be a comment inside this empty block why it’s empty.